.appveyor.yml

```diff
@@ -12,6 +12,11 @@ install:
   - rust-nightly-%TARGET%.exe /VERYSILENT /NORESTART /DIR="C:\Program Files (x86)\Rust"
   - set PATH=%PATH%;C:\Program Files (x86)\Rust\bin
   - if defined MSYS_BITS set PATH=%PATH%;C:\msys64\mingw%MSYS_BITS%\bin
+  -
+  # Work around undefined references to __acrt_iob_func() per
+  # https://github.com/nabijaczleweli/cargo-update/issues/74#issuecomment-376581009
+  - sed -rie "s/#define std([[:alpha:]]+)[[:space:]]+\(__acrt_iob_func\(([[:digit:]]+)\)\)/#define std\1 (\&__iob_func()[\2])/" "C:\msys64\mingw32\i686-w64-mingw32\include\stdio.h" "C:\msys64\mingw64\x86_64-w64-mingw32\include\stdio.h"
```

Comment:
You have to use sed with `-s` to edit multiple files.

Reply:
> You have to use sed with -s to edit multiple files.

Thanks for the tip! Unfortunately, I'm still seeing the same error.